bezpy/mt/io.py

```diff
@@ -189,3 +192,29 @@ def get_1d_site(name):
         return _SITES1D[newname]
 
     raise ValueError("No 1d site profile with the name: " + name)
+
+def read_runinfo(site,root):
+    site.runinfo = {}
+
+    # fieldnotes = xml_site.find("FieldNotes")
+    for field in root.findall("FieldNotes"):
+        # runid of fieldnote
+        runid = field.attrib['run']
+        site.runinfo[runid] = {}
+
+        try:
+            site.NIMSid = get_text( field.find('Instrument'), 'Id')
+        except KeyError:
+            site.NIMSid = None
```

Comment:
Referring to my comment above: If there are some runs without NIMSid then this section could override a previous run's NIMSid.

Reply:
Thanks. That's correct